packager/packager.cc

```diff
@@ -1490,9 +1437,6 @@ void Packager::run(core::GlobalState &gs, WorkerPool &workers, absl::Span<ast::P
     findPackages(gs, files);
     setPackageNameOnFiles(gs, files);
 
-    // Step 2:
-    // * Find package files and rewrite them into virtual AST mappings.
-    // * Find files within each package and rewrite each to be wrapped by their virtual package namespace.
```

Comment:
I got rid of the step numbers because it says step 1a came after step 1, and step 2 comes after both step 1 and step 1a—it doesn't seem useful to number these anymore.

I got rid of these comments, because they talk about doing rewrites that we no longer do (back when we modeled package visibility as a series of AST rewrites)